zendesk_attachment_backup.py

```diff
@@ -69,37 +73,51 @@ def collect_urls(input_files, print_urls, **kwargs):
 
    # Print URLs to file if requested
    if print_urls:
-      with open('attachment_urls.txt', "w") as fh:
+      target_file=os.path.join(kwargs.get("output_dir"), 'attachment_urls.txt')
```

Comment:
I agree that most of it makes sense, but sometimes, I can't help but wonder
...
"Always use 1 whitespace before and after assignments... unless they are
defaults in a prototype, or a keyword arg... unless it's an annotation "

I don't think Python's that hard enough to read in the first place. Then
again I come from Perl, so that might explain a couple of things...
Although to be fair, if following the PEP8 was a requirement here, I would
just suck it up and do it, I just abuse the freedom we have :)